### ISSUE TITLE
Tutorials page

### ISSUE THREAD

**Issue by hao-fang** [acl-org/acl-2020-virtual-conference#24]
**Number of Volunteers**: 2

We have 8 tutorials in total.
Besides the website development, we need to have a volunteer to collect necessary information from tutorial chairs.

**Comment by raosudha89** [comment]
The main page of tutorials should contain the following:
- [ ] Overall schedule of the tutorials (https://acl2020.org/program/tutorials/). There will be several parallel tracks. More information can be found in the July 5th tab of [this spreadsheet](https://docs.google.com/spreadsheets/d/1sTRj-skkp7qYz1zwqvmPuXRay6HIHEOS7NOuUMsW8oE/edit?usp=sharing) 
- [ ] Link to each tutorial

Some tutorials have both live sessions and pre-recorded talks. Whereas some others have only pre-recorded talks. Depending on that, each individual tutorial page should include following:
- [ ] SlidesLive video of pre-recorded talks
- [ ] Zoom link for live session
- [ ] RocketChat link

**Comment by sgdgp** [comment]
@hao-fang @raosudha89  Should I use the template of the "Workshops" page to list out the tutorials ?

**Comment by hao-fang** [comment]
@sgdgp Yes, let's use that template.

**Comment by raosudha89** [comment]
@sgdgp : I am currently working on the tutorials page. I will list out the tutorials and assign specific tasks under it for you. I will tag you here with those tasks. Thanks.

**Comment by raosudha89** [comment]
- [ ] Create RocketChat channel for each tutorial
      @hao-fang: How can I create new channels on RocketChat? Do I have the permission?

**Comment by raosudha89** [comment]
@sgdgp  and @hao-fang 

I have created a basic tutorial page with all tutorial information added to it. Please review https://github.com/acl-org/acl-2020-virtual-conference/pull/109

- [ ]  Next, we need to think of a way to show the schedule of tutorials since some tutorials are occurring twice and some are occurring only once. Check this https://acl2020.org/program/tutorials/

**Comment by hao-fang** [comment]
> * [ ]  Create RocketChat channel for each tutorial
>   @hao-fang: How can I create new channels on RocketChat? Do I have the permission?

@raosudha89 Yes, you have the permission to create channels. Let me know if it doesn't work. I've also sent an email to our committee about RocketChat channels. You can find some explanation there.

**Comment by sgdgp** [comment]
@raosudha89 which task should I focus on in this issue ?

**Comment by raosudha89** [comment]
@sgdgp I think the next task here it to customize each tutorial page (e.g. https://virtual.acl2020.org/tutorial_T1.html),  since not all tutorials have the same format this year. 

Here is the per tutorial information https://docs.google.com/spreadsheets/d/1aUIrybF8vkIMSXLcEReD6jIBHMPy8PGNFj3M3TMIZgc/edit#gid=936853935 

- If a tutorial has pre-recorded video: Add the slideslive embedding code to add the video
- If a tutorial is entirely live: Add a link the zoom session and slideslive embedding that shows the live session
- All tutorials should include RocketChat session. But Maybe we should decrease the size of the RocketChat window?

**Comment by sgdgp** [comment]
@raosudha89 Ok. I will work on this.

**Comment by raosudha89** [comment]
@sgdgp  Is there any update on this? Just checking to see if you need help. We plan to do a dry-run of the website this Friday. So it would be helpful if we could get this done before that.

**Comment by hao-fang** [comment]
I did some changes on the page https://virtual.acl2020.org/tutorial_T1.html
But I didn't deal with the live vs. pre-recorded. We need to check with SlidesLive to see what kind of URL they will provide us for the livestream.

**Comment by sgdgp** [comment]
@hao-fang @raosudha89  Sorry for the delay. I am working on this currently.
I see that the rocketchat rooms for the tutorials are not created yet. I see a reference to create rooms from #38. I will try that. Could you send me the dummy account credentials used to create the chat rooms for papers ?

**Comment by hao-fang** [comment]
@sgdgp I've send the account information to you.

**Comment by sgdgp** [comment]
@hao-fang I created channels for the tutorials. But I mistakenly created the first set with names "tutorial-1" instead of "tutorial-T1". Do I have permission to delete channels ?

**Comment by hao-fang** [comment]
@sgdgp Yes -- you can delete those with the acl2020-admin-bot.

**Comment by sgdgp** [comment]
Done

**Comment by raosudha89** [comment]
@sgdgp 

For each tutorial, we also need a high-level description of how the particular virtual tutorial will be held. This will be useful since there are a lot of variations. Some are entirely on zoom, some have pre-recorded videos + live sessions, while some are entirely pre-recorded. Check the tab July 5 on [this spreadsheet](https://docs.google.com/spreadsheets/d/1sTRj-skkp7qYz1zwqvmPuXRay6HIHEOS7NOuUMsW8oE/edit#gid=261172372) for this information.  

Tagging @hao-fang  incase he has more suggestions.

**Comment by raosudha89** [comment]
Closing this as basic tutorial page is done. Creating a new issue for customizing each tutorial page (https://github.com/acl-org/acl-2020-virtual-conference/issues/216)
